src/Razor/src/Microsoft.VisualStudio.RazorExtension/SyntaxVisualizer/SyntaxVisualizerControl.xaml.cs

```diff
@@ -121,6 +130,34 @@ public void ShowFormattingDocument()
 
         if (_fileUriProvider.TryGet(_activeWpfTextView.TextBuffer, out var hostDocumentUri))
         {
+            if (_languageServerFeatureOptions.UseRazorCohostServer)
+            {
+                var request = new DocumentContentsRequest
```

Comment:
To work around IVT issues, create a static method on `DocumentContentsRequest` that has the signature `Create(Uri hostDocumentUri, GeneratedDocumentKind kind)` that constructs the object.